interop/cpp/interop.cpp

```diff
@@ -112,6 +112,10 @@ int main(int argc, char ** argv) {
         ooe.m_i_str.emplace(5, "five");
         ooe.e = com::microsoft::bond::interop::_bond_enumerators::SomeEnum::B;
         ooe.oos.str = "one of something, anyway";
+        unsigned char blorb_bytes[]{1, 2, 3, 4, 5};
```

Comment:
It's the bytes for `blorb`. `blorb` is just to avoid the reserved word `blob`. Didn't put a ton of thought into a clever name for "the member which is a blob" because we aren't going to merge `interop/`.